### PR TITLE
Add JavaScript export question 

### PR DESCRIPTION
**Pull request by richardoey**
Checklist:

- [X] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [X] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

### CODE REVIEW

File: src/data/javascript-quiz.ts [hunk 1]
```diff
@@ -1739,21 +1738,30 @@ const javascriptQuiz = [
...
+  {
+    Question:
+      "In JavaScript, which keyword to make a variable, function, class or object accessible to other module?",
+    Answer: "export",
+    Distractor1: "import",
+    Distractor2: "as",
+    Distractor3: "from",
+    Explanation:
+      "Before we can get access to module features, the first thing we need to do is 'export' them.",
+    Link: "https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules#exporting_module_features"
```
Comment (jdwilkin4) [inline src/data/javascript-quiz.ts:1763]:
```suggestion
    Link: "https://www.freecodecamp.org/news/javascript-modules-explained-with-examples/#"
```

File: src/data/javascript-quiz.ts [hunk 2]
```diff
@@ -1739,21 +1738,30 @@ const javascriptQuiz = [
...
+    Distractor3: "Promise.reject() and Promise.reject()",
     Explanation:
       "Promise.all() and Promise.allSettled() accept an iterable of promises and return a single array of results.",
-    Link:
-      "https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise",
+    Link: "https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise"
   },
+  {
+    Question:
+      "In JavaScript, which keyword to make a variable, function, class or object accessible to other module?",
```
Comment (jdwilkin4) [inline src/data/javascript-quiz.ts:1756]:
```suggestion
      "In JavaScript, which keyword is used to make portions of code accessible to other modules.?",
```

File: src/data/javascript-quiz.ts [hunk 3]
```diff
@@ -1739,21 +1738,30 @@ const javascriptQuiz = [
...
+    Answer: "export",
+    Distractor1: "import",
+    Distractor2: "as",
+    Distractor3: "from",
+    Explanation:
+      "Before we can get access to module features, the first thing we need to do is 'export' them.",
```
Comment (jdwilkin4) [inline src/data/javascript-quiz.ts:1762]:
```suggestion
      "The export keyword is used to make it possible to access small portions of code in other modules",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by richardoey** [comment]
I have changes based on your feedback. Thanks a lot! There are lot of grammar mistakes from me before. Thank you for giving your time checking it. I also has resolved the conflict after pulling from main branch

**Review by jdwilkin4 — APPROVED** [review]
Congrats on another contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**